buildconfig/Jenkins/jenkins-agent.sh

```diff
@@ -91,12 +96,12 @@ elif [ -f ${HOME}/Jenkins/${JAR_FILE} ]; then
 else
   JAR_FILE_TMP=/tmp/${JAR_FILE}
   if [ ! -f ${JAR_FILE_TMP} ]; then
-    echo "Downloading slave jar file to ${JAR_FILE_TMP}"
+    echo "Downloading agent jar file to ${JAR_FILE_TMP}"
     if [ $(command -v curl) ]; then
-      echo "curl --location -o ${JAR_FILE_TMP} ${JENKINS_URL}/jnlpJars/${JAR_FILE}"
-      curl -o ${JAR_FILE_TMP} ${JENKINS_URL}/jnlpJars/${JAR_FILE}
+      echo "curl --location -o ${JAR_FILE_TMP} ${JENKINS_REPO_URL}/${JAR_VERSION}/${JAR_FILE}"
```

Comment:
Just for my knowledge, could you explain why are you replacing the old link with one directly from the Jenkins repo?

Reply:
In the past, the Jenkins agents provided via our Jenkins server were always compatible with the version of Jenkins we were running. For some reason, this is now not the case. We have to go directly to the Jenkins repo and find an appropriate version there.

This could perhaps be because Jenkins are moving away from distributing agents in this way, and the preferred solution is to use agents baked into docker containers (doesn't help us on mac!).

Reply:
Ok, thanks for the explanation